duckduckgo/macos-browser
# Phishing Detection Error Page Scheme Handler

Task/Issue URL: https://app.asana.com/0/1199230911884351/1208149630394250/f
Tech Design URL: https://app.asana.com/0/481882893211075/1207220724600204
CC:

**Description**:
Implement duck://error?kind=phishing handler so error pages correctly set tab.error, and load the relevant error page.

**Steps to test this PR**:
1. Visit privacy-test-pages.site/security/badware/phishing.html
2. Ensure phishing error page is shown.



**Definition of Done**:


---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
